go/services/reports/internal/brokers/report_requested.go

```diff
@@ -0,0 +1,60 @@
+package brokers
+
+import (
+	"os"
+
+	"github.com/Magpie-Monitor/magpie-monitor/pkg/envs"
+	"github.com/Magpie-Monitor/magpie-monitor/pkg/message-broker"
+	"github.com/Magpie-Monitor/magpie-monitor/services/reports/pkg/insights"
+	"go.uber.org/fx"
+	"go.uber.org/zap"
+)
+
+var REPORT_REQUESTED_TOPIC_KEY = "REPORT_REQUESTED_BROKER_TOPIC"
+
+type ReportRequest struct {
+	ClusterId                *string                                     `json:"clusterId"`
+	SinceMs                  *int64                                      `json:"sinceMs"`
+	ToMs                     *int64                                      `json:"toMs"`
+	ApplicationConfiguration []*insights.ApplicationInsightConfiguration `json:"applicationConfiguration"`
+	NodeConfiguration        []*insights.NodeInsightConfiguration        `json:"nodeConfiguration"`
+	MaxLength                *int                                        `json:"maxLength"`
+}
+
+type ReportRequested struct {
+	CorrelationId string        `json:"correlationId"`
```

Comment:
Is CorrelationId same as RequestId?

Reply:
It was the way we initially intented to use it. CorrelationId is a more fitting name considering what it will be used for. Here is an example from octa on what it is: https://www.okta.com/identity-101/correlation-id/